relay-event-normalization/src/normalize/mod.rs

```diff
@@ -412,7 +412,7 @@ fn get_metric_measurement_unit(measurement_name: &str) -> Option<MetricUnit> {
         "frames_frozen" => Some(MetricUnit::None),
         "frames_frozen_rate" => Some(MetricUnit::Fraction(FractionUnit::Ratio)),
         "time_to_initial_display" => Some(MetricUnit::Duration(DurationUnit::MilliSecond)),
-        "time_to_first_display" => Some(MetricUnit::Duration(DurationUnit::MilliSecond)),
+        "time_to_full_display" => Some(MetricUnit::Duration(DurationUnit::MilliSecond)),
```

Comment:
Are we ingesting this measurement already? Should we support both to keep compatibility with old SDKs?

Reply:
We're ingesting `time_to_full_display` already! We never had a `time_to_first_display` measurement, I think it was just accidentally added as `time_to_first_display` here (cc: @wmak can confirm!)